rust/lance-index/src/scalar/bloomfilter.rs

```diff
@@ -52,10 +52,11 @@ const BLOOMFILTER_INDEX_VERSION: u32 = 0;
 #[derive(Debug, Clone)]
 struct BloomFilterStatistics {
     fragment_id: u64,
-    // zone_start is the start row of the zone in the fragment, also known
-    // as local row offset
+    // zone_start is the actual first row address (local offset within fragment)
```

Comment:
Local offset and row address are **not** the same thing.
```suggestion
    // zone_start is the start row of the zone in the fragment, also known
    // as local row offset. To get the first row address, you can do 
    // `fragment_id << 32 + zone_start`.
```

Reply:
true, will fix the wrong wording